src/pip/_internal/vcs/subversion.py

```diff
@@ -63,7 +63,7 @@ def export(self, location):
         """Export the svn repository at the url to the destination location"""
         url, rev = self.get_url_rev()
         rev_options = get_rev_options(self, url, rev)
-        url = self.remove_auth_from_url(url)
+        url = remove_auth_from_url(url)
```

Comment:
this removes the auth information that will later be used in the command :-1:

Reply:
It's the behavior it had prior to this patch; I think that's fine.